_includes/v22.2/misc/delete-statistics.md

```diff
@@ -2,16 +2,14 @@ To delete statistics for all tables in all databases:
 
 {% include_cached copy-clipboard.html %}
 ~~~ sql
-> DELETE FROM system.table_statistics WHERE true;
+DELETE FROM system.table_statistics WHERE true;
 ~~~
 
 To delete a named set of statistics (e.g, one named "users_stats"), run a query like the following:
 
 {% include_cached copy-clipboard.html %}
 ~~~ sql
-> DELETE FROM system.table_statistics WHERE name = 'users_stats';
+DELETE FROM system.table_statistics WHERE name = 'users_stats';
 ~~~
 
-After deleting statistics, restart the nodes in your cluster to clear the statistics caches.
```

Comment:
cc @rytaft this advice is out-of-date, correct?

Reply:
Not that I know of -- do you know what change fixed this? Is it due to the fact that we're using rangefeeds instead of Gossip?

Reply:
Correct. With https://github.com/cockroachdb/cockroach/pull/62563 a deletion of a stats row will trigger the range feed handle in `stats_cache.go`, and the timestamp of the delete rangefeed event will differ from the cache's `lastTS`:

https://github.com/cockroachdb/cockroach/blob/3bc0992225673c2e8c80288a5c619d78f8b928cd/pkg/sql/stats/stats_cache.go#L155

So a stats refresh should be triggered.

Reply:
Makes sense - thanks!